src/cclib/parser/gamessparser.py

```diff
@@ -1371,7 +1371,7 @@ def extract(self, inputfile, line):
             assert coords_and_charge.split()[-1] == '(A.U.)'
             reference = numpy.array([float(x) for x in coords_and_charge.split()[:3]])
             reference = utils.convertor(reference, 'bohr', 'Angstrom')
-            charge = float(coords_and_charge.split()[-2])
+            charge = int(float(coords_and_charge.split()[-2]))
```

Comment:
I think there should be a ```round``` here. If the string is something like ```1.9999```, then the charge will incorrectly be determined as ```1```: